frontend/heart-code/src/components/profile-page/Profile.tsx

```diff
@@ -23,7 +23,7 @@ export default function Profile({ user, isPersonalPage }: Props) {
         <Card.Body className="card-content">
           <Card.Title>{user.alias}</Card.Title>
           <Card.Subtitle className="card-content">
-            [age] {convertGender(user.gender)}
```

Comment:
`{convertGender(user.gender)}, {user.age}` zou je hier voor de duidelijk `age: {user.age}` van kunnen maken?

Reply:
ik vind het erg overbodig om dat er bij te zetten, bij algemene informatie worden dat soort dingen er meestal niet bij gezet. hoewel op veel plekken de leeftijd vaak achter de naam staat. maar eigenlijk altijd zonder "age: [#]"

Reply:
is goed